Bin/Snapshot.php

```diff
@@ -49,19 +49,46 @@
  */
 class Snapshot extends Console\Dispatcher\Kit
 {
+    /**
+     * User friendly labels for changelog themes.
+     *
+     * @const
```

Comment:
I'm pretty sure this annotation is useless :-p.

Reply:
Of course 😄.